tests/containers/aboutTheData/UnlinkedDataContainer-test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, waitFor } from 'test-utils';
+import * as aboutTheDataHelper from 'helpers/aboutTheDataHelper';
+import UnlinkedDataContainer from 'containers/aboutTheData/modals/UnlinkedDataContainer';
+
+const defaultProps = {
+    agencyData: {
+        fiscalYear: 2020,
+        fiscalPeriod: 8,
+        agencyCode: '012'
+    }
+};
+
+describe('Unlinked Data Container', () => {
+    it('should call api one time on mount', () => {
+        const unlinkedDataRequest = jest.spyOn(aboutTheDataHelper, 'fetchMockUnlinkedData');
+        render(<UnlinkedDataContainer {...defaultProps} />);
+        waitFor(() => expect(unlinkedDataRequest).toHaveBeenCalledTimes(1));
```

Comment:
nice. this is how i've done these tests. One thing to share about them, I suspect this may evaluate to true even if it renders twice. It would waitFor it to be called once, then exit the test as true, when it could call it again.

Any thoughts on this?

Reply:
Yes, this does not work. I set three calls and it still passed. Removing the waitFor seems to work.